### ISSUE TITLE
Day-of-the-week relative shifting failed

### ISSUE THREAD

**Issue by zero-master** [crsmithdev/arrow#489]
I am not able to run the example listed on this page: http://arrow.readthedocs.io/en/latest/

```python
Python 3.6.2 (default, xxxxx) 
[GCC 7.1.1 20170630] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import arrow
>>> arw = arrow.utcnow()
>>> arw
<Arrow [2017-11-12T17:05:52.334182+00:00]>
>>> arw.shift(weekday=5)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/Projects/app/env/lib/python3.6/site-packages/arrow/arrow.py", line 461, in shift
    raise AttributeError()
AttributeError
>>> 
```

```#
pip freeze | grep arrow
arrow==0.10.0
```

**Comment by AlJohri** [comment]
duplicate of #450

**Comment by systemcatch** [comment]
fixed in #389
